oozie-to-airflow/converter/primitives.py

```diff
@@ -79,29 +98,38 @@ def __eq__(self, other):
 class Task:  # pylint: disable=too-few-public-methods
     """Class for Airflow Task"""
 
-    task_id: str
-    template_name: str
-    template_params: Dict[str, Any]
-
-    def __init__(self, task_id, template_name, trigger_rule=TriggerRule.DUMMY, template_params=None):
-        self.task_id = task_id
-        self.template_name = template_name
+    def __init__(
+        self,
+        task_id: str,
+        template_name: str,
+        trigger_rule=TriggerRule.DUMMY,
+        template_params: Dict[str, Any] = None,
+    ):
+        self.task_id: str = task_id
+        self.template_name: str = template_name
         self.trigger_rule = trigger_rule
-        self.template_params = template_params or {}
+        self.template_params: Dict[str, Any] = template_params or {}
+
+    @property
+    def task_variable_name(self):
+        return convert_to_python_variable(self.task_id)
 
     @property
     def rendered_template(self):
         return render_template(
             template_name=self.template_name,
             task_id=self.task_id,
             trigger_rule=self.trigger_rule,
+            task_variable_name=self.task_variable_name,
             **self.template_params,
         )
 
     def __repr__(self) -> str:
         return (
-            f'Task(task_id="{self.task_id}", template_name="{self.template_name}", '
-            f'trigger_rule="{self.trigger_rule}", template_params={self.template_params})'
+            f'Task(task_id="{self.task_id}", '
+            f'template_name="{self.template_name}", '
+            f'trigger_rule="{self.trigger_rule}", '
+            f"template_params={self.template_params}, "
```

Comment:
```suggestion
            f"template_params={self.template_params})"
```
Probably a typo.

Reply:
Nope. It's as intended - they are concatenated and displayed in one line separated with ", "

Reply:
``Task(task_id="test-id-prepare", template_name="prepare.tpl", trigger_rule="dummy", template_params={'prepare_command': '$DAGS_FOLDER/../data/prepare.sh -c my-cluster -r europe-west3 -d "{}"', 'prepare_arguments': ['{CTX["nameNode"]}/examples/mapreduce/output']},``

Is this end of code correct?